mediorum/server/upload_client.go

```diff
@@ -0,0 +1,78 @@
+package server
+
+import (
+	"fmt"
+	"time"
+
+	"gorm.io/gorm/clause"
+)
+
+func (ss *MediorumServer) startUploadScroller() {
+	time.Sleep(time.Minute)
+
+	for {
+		for _, peer := range ss.Config.Peers {
+			if peer.Host == ss.Config.Self.Host {
+				continue
+			}
+
+			uploadCursor := &UploadCursor{
+				Host: peer.Host,
+			}
+
+			// load prior cursor for host
+			ss.crud.DB.First(&uploadCursor, "host = ?", peer.Host)
```

Comment:
wouldn't this reset the Host field in uploadCursor to an empty string if no db record is found (ie the first time this runs)?